Pipeline/controllers/RnaMigosController.py

```diff
@@ -1,9 +1,13 @@
 # This is where the "business logic" of the pipeline will live for the TwoAdder service
```

Comment:
i know this wasn't you but remove twoadder from the comment, it's leftover from copy pasting lol

Reply:
Thanks for pointing that out